pegasus/sites.v3/code.org/public/educate/csp.redirect

```diff
@@ -0,0 +1 @@
+https://code.org/curriculum/csp
```

Comment:
There is some weird legacy stuff going on with this URL so I pointed this to the full url to make it work.